### PR TITLE
Login init backend

### PR DESCRIPTION
**Pull request by Asamya**

### CODE REVIEW

File: backend/src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+spring.data.mongodb:
```
Comment (benjamin-ranft) [inline backend/src/main/resources/application.yml:1]: you could keep the application.yml empty and set an environment variable in intelij instead.

SPRING_DATA_MONGODB_DATABASE    and "rubbleDB" as the value

File: backend/src/test/java/de/neuefische/rubble/controller/LoginControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        assertThat(response.getStatusCode(), is(HttpStatus.OK));
+
+        String token = response.getBody();
+        Claims claims = Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody();
+
+        assertThat(claims.getSubject(), is("walther"));
+        assertThat(claims.getExpiration().after(new Date()), is(true));
+    }
+    
+     */
```
Comment (benjamin-ranft) [inline backend/src/test/java/de/neuefische/rubble/controller/LoginControllerTest.java:71]: Here is the test I wrote for mine, should be working for you too if you adjust the details

`@Test
    public void loginWithValidCredentialsShouldReturnJwtToken() {

        //GIVEN
        LoginDto loginDto = new LoginDto(
                "ben@test.com",
                "password"
        );

        //WHEN
        ResponseEntity<String> response = restTemplate.postForEntity("http://localhost:" + port + "/auth/login", loginDto, String.class);

        //THEN
        assertThat(response.getStatusCode(), is(HttpStatus.OK));

        String token = response.getBody();
        Claims claims = Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody();

        assertThat(claims.getSubject(), is("ben@test.com"));
        assertThat(claims.getExpiration().after(new Date()), is(true));

    }`

**Review by benjamin-ranft — APPROVED** [review]
Once test fixed, ready to go!

File: backend/src/test/java/de/neuefische/rubble/security/JwtAuthFilterTest.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    private final String secretKey = "somesecret";
+
+
+   /* @Test
+    @DisplayName("Valid token should display 200 Ok")
+    public void getWithValidJwtTokenShouldReturn200Ok(){
```
Comment (fabianschmauder) [inline backend/src/test/java/de/neuefische/rubble/security/JwtAuthFilterTest.java:37]: why?

**Review by fabianschmauder — COMMENTED** [review]
your test is commented out 😢

**Pull request merged**